.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,10 @@
+<!-- Please use this template only for submitting feature requests or enhancements -->
+
+/kind feature
+
+
+**What would you like to add**
```

Comment:
Two points here:

1. I think it would be useful to have an alternate wording of "what happened" and "what you expected to happen" because it holds true for feature/enhancement requests too. Most feature requests start out as "I did X and was expecting Y, but I got Z. I think it would be cool to add a feature to get the output as Z". Getting output as Z is not always a bug, it could also mean adding a new feature.

2. Would prefer this question as `What would you like to be added` or something like that. Usually the person submitting the feature request is not the one implementing it. ;)